warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -338,6 +338,12 @@ public int run(String[] args) throws Exception {
                 configureBulkPartitionerAndOutputFormatter(job, cbHelper, conf, outputFs);
             } catch (Exception e) {
                 log.error(e);
+                log.error("Deleting orphaned directory: " + workDirPath);
```

Comment:
I'm wondering if this should be an INFO since this line itself is not an error. Usually when you see ERROR in the log, it's something you need to investigate (like the error logged above). This is more just telling us we're cleaning up due to a previous error.